rllib/policy/dynamic_tf_policy_v2.py

```diff
@@ -486,7 +492,7 @@ def _init_input_dict_and_dummy_batch(
         # Placeholder for `is_training` flag.
         self._input_dict.set_training(self._get_is_training_placeholder())
 
-        return timestep, explore
+        return self.timestep, self.explore
```

Comment:
why bother returning these if we are not using them in the constructor?

Reply:
Thanks, changed.